hack/conformance.sh

```diff
@@ -1,4 +1,4 @@
-set -xe
+#!/bin/bash -xe
```

Comment:
Can we use `set -euxo pipefail` ? 

So we also fail on pipes failures ?

Reply:
done, thx